example/glue/ScavengerDelegate.hpp

```diff
@@ -131,10 +131,12 @@ class MM_ScavengerDelegate : public MM_BaseVirtual {
 	 * @param[in] objectPtr The object to be scanned
 	 * @param[in] allocSpace Space for in-place instantiation of scanner
 	 * @param[in] flags See GC_ObjectScanner::InstanceFlags. One of scanRoots or scanHeap will be set , but not both.
+	 * @param[in] reason See MM_ScavengeScanReason
+	 * @param[out]shouldRemember
 	 * @return Pointer to object scanner, or NULL if object not to be scanned (eg, leaf object).
 	 * @see GC_ObjectScanner
 	 */
-	GC_ObjectScanner *getObjectScanner(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, void *allocSpace, uintptr_t flags);
+	GC_ObjectScanner *getObjectScanner(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, void *allocSpace, uintptr_t flags, MM_ScavengeScanReason reason, bool *shouldRemember);
```

Comment:
don't you need to leave the old API temporarily and remove it later after delegates in downstream projects are updated ?

Reply:
in that case, we need 2 merges in omr and 2 merges in openJ9(at least one merges)